test/hpr_gwmp_router_SUITE.erl

```diff
@@ -355,6 +355,11 @@ gateway_dest_redirect(_Config) ->
     USPacketUp = fake_join_up_packet(),
     EUPacketUp = USPacketUp#packet_router_packet_up_v1_pb{hotspot = PubKeyBin, region = 'EU868'},
```

Comment:
Field name 'hotspot' will change to 'gateway' when PR #41 is merged into 'main'.